docs/general/userstyles.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Userstyles in AdGuard Ad Blocker (Experimental)
+sidebar_position: 9
+---
+
+Userstyles allow users to customize their online experience. Whether you’re looking to change the appearance of a website or automate repetitive tasks, styles offer a world of possibilities.
+
+AdGuard has the option to upload or create your own userstyles. This is an advanced user experience, so you will need some knowledge of HTML and CSS.
+
+:::info Supported apps
+
+Currently, two AdGuard apps allow to create and manage userstyles: AdGuard for Windows and AdGuard for Mac.
+
+:::
+
+This is an experimental feature, so if you encounter any problems while adding or creating the userstyle, please contact our [support team](support@adguard.com).
```

Comment:
Exhaustive list of all broken links found:

- On source page path = /KnowledgeBase/general/userstyles/:
   -> linking to support@adguard.com/ (resolved as: /KnowledgeBase/general/userstyles/support@adguard.com/)